coap-gateway/service/auth.go

```diff
@@ -99,37 +98,37 @@ func verifyChain(chain []*x509.Certificate, capool *x509.CertPool) (string, erro
 		}
 	}
 	if !ekuHasClient {
-		return "", fmt.Errorf("not contains ExtKeyUsageClientAuth")
+		return fmt.Errorf("not contains ExtKeyUsageClientAuth")
```

Comment:
the extended key usage field in the device certificate does not contain client authentication

Reply:
Fixed